CDN/CloudFront.php

```diff
@@ -162,7 +162,7 @@ private function getClient()
     /**
      * For testing only.
      *
-     * @param $client
+     * @param CloudFrontClient $client
```

Comment:
Should be correct, as the CloudFrontClient::factory method returns a CloudFrontClient.
I'll create a seperate PR for the type check.